bin/grommet.js

```diff
@@ -43,8 +43,8 @@ var tasks = {
         process.chdir('./' + app);
 
         var templateFolder = path.join(grommetPath, 'templates/init/**');
-        var mobileIcon = path.join(grommetPath, 'mobile-app-icon.png');
-        var shortcutIcon = path.join(grommetPath, 'shortcut-icon.png');
+        var mobileIcon = path.join(grommetPath, 'src/img/mobile-app-icon.png');
```

Comment:
> I believe this should be img/mobile-app-icon.png. When we install grommet, there is no src anymore.

After run *grommet init*, all source files should go the target *src* directory (e.g. *app-name/src/*), so the *index.html* located under *app-name/src/index.html* can reach them. Isn't that right?

$ tree app-name/
app-name/
├── customEslintrc.json
├── grommet-toolbox.config.js
├── gulpfile.babel.js
├── package.json
└── src
    ├── img
    │   ├── mobile-app-icon.png
    │   └── shortcut-icon.png
    ├── index.html
    ├── js
    │   ├── components
    │   │   └── TodoAppDashboard.js
    │   └── index.js
    └── scss
        └── index.scss